eventsourcing/src/main/java/org/axonframework/eventsourcing/AbstractSnapshotter.java

```diff
@@ -217,6 +236,17 @@ public Builder executor(Executor executor) {
             return this;
         }
 
+        /**
+         * Sets the {@link SpanFactory} implementation to use for providing tracing capabilities.
```

Comment:
Builder methods their JavaDoc generally state the default setting in absence of invocation if there's a default setting.